src/Form/ThirdPartyServicesForm.php

```diff
@@ -34,85 +34,61 @@ public function buildForm(array $form, FormStateInterface $form_state, array &$i
-      '#title' => $this->t('Google Analytics Web Property ID'),
-      '#type' => 'textfield',
-    ];
+    if (\Drupal::moduleHandler()->moduleExists('google_analytics')) {
+      $form['google_analytics_account'] = [
+        '#default_value' => $ga_config->get('account'),
+        '#description' => $this->t('This ID is unique to each site you want to track separately, and is in the form of UA-xxxxxxx-yy. To get a Web Property ID, <a href=":analytics">register your site with Google Analytics</a>, or if you already have registered your site, go to your Google Analytics Settings page to see the ID next to every site profile. <a href=":webpropertyid">Find more information in the documentation</a>.', [':analytics' => 'http://www.google.com/analytics/',
+          ':webpropertyid' => Url::fromUri('https://developers.google.com/analytics/resources/concepts/gaConceptsAccounts', ['fragment' => 'webProperty'])
+            ->toString()
+        ]),
+        '#maxlength' => 20,
+        '#placeholder' => 'UA-',
+        '#title' => $this->t('Google Analytics Web Property ID'),
+        '#type' => 'textfield',
+      ];
+    }
 
     // Google Tag Manager Account ID.
-    $form['google_tag_manager_id'] = [
-      '#title' => $this->t('Google Tag Manager Container ID'),
-      '#description' => $this->t('The ID assigned by Google Tag Manager (GTM) for this website container. To get a container ID, <a href="https://tagmanager.google.com/">sign up for GTM</a> and create a container for your website.'),
-      '#default_value' => $gtm_config->get('container_id'),
-      '#attributes' => ['placeholder' => ['GTM-xxxxxx']],
-      '#maxlength' => 20,
-      '#type' => 'textfield',
-    ];
+    if (\Drupal::moduleHandler()->moduleExists('google_tag')) {
+      $form['google_tag_manager_id'] = [
+        '#title' => $this->t('Google Tag Manager Container ID'),
+        '#description' => $this->t('The ID assigned by Google Tag Manager (GTM) for this website container. To get a container ID, <a href="https://tagmanager.google.com/">sign up for GTM</a> and create a container for your website.'),
+        '#default_value' => $gtm_config->get('container_id'),
+        '#attributes' => ['placeholder' => ['GTM-xxxxxx']],
+        '#maxlength' => 20,
+        '#type' => 'textfield',
+      ];
+    }
 
     // Optimizely ID Number.
-    $form['optimizely_id'] = [
-      '#type' => 'textfield',
-      '#placeholder' => 'xxxxxxxxxx',
-      '#title' => $this->t('Optimizely ID Number'),
-      '#default_value' => $optimizely_config->get('optimizely_id'),
-      '#description' => $this->t('Your Optimizely account ID.</br>In order to use this module, you\'ll need an <a href="http://optimize.ly/OZRdc0" target="_blank">Optimizely account</a>. </br>See <a href="https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Optimizely.md" target="_blank">Open Y documentation</a> for Optimizely.'),
-    ];
-
-    // Recaptcha keys.
-    $form['recaptcha'] = [
```

Comment:
Let's keep recaptcha but with the same condition, as others. If module recaptcha is installed then show settings.